schema/site_metadata.json

```diff
@@ -50,6 +58,59 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
```

Comment:
I don't think this formulation here is quite right -- why would you ever have more than one specification for area?  This should either be one field with a canonical type ("area_m2") or two separate fields "area_units" and "area_value" or something... Or just "area" and we can default the units to m2 without an explicit field, and then add in a "area_units" or something like that later.

The goal here is to prevent potential ambiguity in the source specification!